packages/analytics/src/helpers.ts

```diff
@@ -24,10 +24,35 @@ import {
 import { DynamicConfig, DataLayer, Gtag, MinimalDynamicConfig } from './types';
 import { GtagCommand, GTAG_URL } from './constants';
 import { logger } from './logger';
+import { AnalyticsError, ERROR_FACTORY } from './errors';
 
 // Possible parameter types for gtag 'event' and 'config' commands
 type GtagConfigOrEventParams = ControlParams & EventParams & CustomParams;
 
+// Create a TrustedTypes policy that we can use for updating src
+// properties
+let _ttPolicy: Partial<TrustedTypePolicy>;
```

Comment:
If this is going to be used by multiple products we should probably move it to app at some point, but that can be done next time another product's use case pops up. As mentioned above, this can probably all be moved inside insertScriptTag or even into its own function which is then called in insertScriptTag (which we can pull out and implement in app later). This will make testing both cases (window.trustedTypes exists, or it doesn't) easier.